templates/admin/release_report_detail.html

```diff
@@ -8,6 +8,20 @@
 .link-icons a:hover {
   color: var(--primary-color);
 }
+
+.sponsor-message ul > li {
+  list-style: circle;
+  margin-left: 2rem;
+
+}
+.sponsor-message a {
+  color: rgb(2, 132, 199);
+}
+
+.sponsor-message a:hover {
+  color: rgb(255, 159, 0);
+}
```

Comment:
any styling for the sponsor message will need to be added here, or perhaps directly in the html in the `sponsor_message` in the Version model